lib/web_ui/test/canvaskit/embedded_views_test.dart

```diff
@@ -313,43 +316,95 @@ void testMain() {
         dispatcher.rasterizer!.draw(sb.build().layerTree);
       }
 
-      int countCanvases() {
-        return flutterViewEmbedder.sceneElement!.querySelectorAll('canvas').length;
-      }
-
       // Frame 1:
       //   Render: up to cache size platform views.
       //   Expect: main canvas plus platform view overlays.
-      renderTestScene(viewCount: configuration.canvasKitMaximumSurfaces);
-      expect(countCanvases(), configuration.canvasKitMaximumSurfaces);
+      renderTestScene(viewCount: 8);
+          expectSceneMatches(<_EmbeddedViewMarker>[
```

Comment:
The whitespace is a little bit odd (+4?) in this expect.

Reply:
done